packages/liveblocks-node/src/client.ts

```diff
@@ -194,6 +294,379 @@ export class Liveblocks {
     }
   }
 
+  /* -------------------------------------------------------------------------------------------------
+   * Room
+   * -----------------------------------------------------------------------------------------------*/
+
+  /**
+   * Returns a list of your rooms. The rooms are returned sorted by creation date, from newest to oldest. You can filter rooms by metadata, users accesses and groups accesses.
+   * @param params.limit (optional) A limit on the number of rooms to be returned. The limit can range between 1 and 100, and defaults to 20.
+   * @param params.startingAfter (optional) A cursor used for pagination. You get the value from the response of the previous page.
+   * @param params.userId (optional) A filter on users accesses.
+   * @param params.metadata (optional) A filter on metadata. Multiple metadata keys can be used to filter rooms.
+   * @param params.groupIds (optional) A filter on groups accesses. Multiple groups can be used.
+   * @returns A list of rooms.
+   */
+  public async getRooms(
+    params: {
+      limit?: number;
+      startingAfter?: number;
+      metadata?: RoomMetadata;
+      userId?: string;
+      groupIds?: string;
+    } = {}
+  ): Promise<{
+    nextPage: string | null;
+    data: RoomInfo[];
+  }> {
+    const path = url`v2/rooms`;
```

Comment:
It will work, but let's make all URL paths absolute everywhere, by letting them all start with a `/`. It also helps hugely with the greppability of the codebase if we do that:

```suggestion
    const path = url`/v2/rooms`;
```

Reply:
Agreed. Nice to keep it consistent and makes sense to make this an absolute url.